infrastructure/k8s/services/secrets.yaml

```diff
@@ -0,0 +1,29 @@
+project_name: "taskcluster-secrets"
+secrets:
+  azure_account: ".Values.tfAzureAccount"
+  azure_crypto_key: ".Values.azureCryptoKey"
+  azure_signing_key: ".Values.azureSigningKey"
```

Comment:
Can you rename these to secretsAzureCryptoKey and secretsAzurSigningKey? I think each service has their own.

Also, can you add generating them to create-random-secrets.sh ?

Reply:
I'm adding all the secret generation stuff in a separate PR to avoid the mother of all merge conflicts that would arise from changing the same line in each of the 10 service PRs. Names fixed.